src/webaudio-media.js

```diff
@@ -0,0 +1,56 @@
+import MediaElement from './mediaelement';
+
+/**
+ * WebAudioMedia backend
```

Comment:
can you add a description for the backend here (possibly copied from the discussion on this PR)?